src/dmd/mtype.h

```diff
@@ -815,6 +815,8 @@ class TypeClass : public Type
 class TypeTuple : public Type
 {
 public:
+    static TypeTuple *empty;    // don't modify!
```

Comment:
Needs a better comment here.

Reply:
FWIW, changed to `'logically immutable' cached global - don't modify (neither pointer nor pointee)!`.

Reply:
I was thinking more along the lines of *what* it stores, rather than the why, but ok.

Reply:
I thought its name would make that clear.